debugging/netcoreapp/debuggerClient.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import { PlatformType } from "./osProvider";
+import { VsDbgClient } from "./vsdbgClient";
+
+export interface DebuggerClient {
+    getDebugger(os: PlatformType): Promise<string>;
+}
+
+export class DefaultDebuggerClient {
+    private static debuggerVersion: string = 'vs2017u5';
+    private static debuggerLinuxRuntime: string = 'debian.8-x64';
+    private static debuggerWindowsRuntime: string = 'win7-x64';
```

Comment:
I thought Windows wasn't supported...?

Reply:
It's not...*quite*.  There *is* a Windows version of `vsdbg` (hence the constant) but isn't currently supported on the base Windows .NET Core images; the maintainers state that `msvsmon` is still the PoR for Windows container debugging as it "supports more debugging scenarios", but for which VS Code does not have an adaptor (AFAIK).